packages/wrangler/src/workflows/commands/instances/send-event.ts

```diff
@@ -0,0 +1,82 @@
+import { fetchResult } from "../../../cfetch";
+import { createCommand } from "../../../core/create-command";
+import { logger } from "../../../logger";
+import { requireAuth } from "../../../user";
+import type { Instance } from "../../types";
+
+export const workflowsInstancesSendEventCommand = createCommand({
+	metadata: {
+		description: "Send an event to a workflow instance",
+		owner: "Product: Workflows",
+		status: "stable",
+	},
+
+	positionalArgs: ["name", "id"],
+	args: {
+		name: {
+			describe: "Name of the workflow",
+			type: "string",
+			demandOption: true,
+		},
+		id: {
+			describe:
+				"ID of the instance - instead of an UUID you can type 'latest' to get the latest instance and send an event to it",
+			type: "string",
+			demandOption: true,
+		},
+		type: {
+			describe: "Type of the workflow event",
+			type: "string",
+			demandOption: true,
+		},
+		payload: {
+			describe:
+				'JSON string for the workflow event (e.g., \'{"key": "value"}\')',
+			type: "string",
+			demandOption: false,
+			default: "{}",
+		},
+	},
+
+	async handler(args, { config }) {
+		const accountId = await requireAuth(config);
+
+		let id = args.id;
```

Comment:
I think it would make a lot of sense to have

```ts
import { retrieveIdFromArg } from "<some number of ../>utils.ts";

const id = await retrieveIdFromArg(args.id);
```

Instead of repeating the same code for all the commands